v4/export/config.go

```diff
@@ -117,6 +129,53 @@ func (conf *Config) GetDSN(db string) string {
 	return dsn
 }
 
+func (config *Config) ExternalStorage(ctx context.Context) storage.ExternalStorage {
+	config.externalStorageOnce.Do(func() {
```

Comment:
Can we remove `externalStorageOnce` and move the create part to [adjustConfig](https://github.com/pingcap/dumpling/blob/48b61dc62c7e27c99b81d939e4614f07e3f3a9d2/v4/export/prepare.go#L73)?

Reply:
@kennytm WDYT?

Reply:
+1

Reply:
Good idea!